bundle/edu.gemini.util.skycalc/src/main/java/edu/gemini/skycalc/TwilightBoundType.java

```diff
@@ -15,8 +15,12 @@
 
     private static final long serialVersionUID = 1;
 
+    // Horizon geometric correction from p. 24 of the Skycalc manual: sqrt(2 * elevation / Re) (radians)
+    static double Re = 6378140; // Radius of the Earth (meters)
+    static double SiteElevation = 4213; // Mauna Kea elevation (meters)
+
     public static final TwilightBoundType OFFICIAL =
-            new TwilightBoundType("Official", 50.0/60.0);
+            new TwilightBoundType("Official", 50.0/60.0 + Math.sqrt(2.0 * SiteElevation / Re) * 180./Math.PI);
```

Comment:
I don't think this belongs here.  I'm not sure where the "Official" twilight definition comes from, but in general these are supposed to be site/altitude definitions of twilight that should be adjusted for altitude elsewhere.  For example, perhaps the `TwilightBoundedNight` calculation needs to take this into account if it isn't being done in skycalc.

Reply:
Something may need to be parameterized by site but we certainly don't want to hardcode it to MK do we?